### PR TITLE
Using perkstore - screenshots missing

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/UsingPerkStore.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.. _UsingPerkStore:
+
+####################
+Using the Perkstore
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
Using the Perk Store
```

File: docs/UsingPerkStore.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+####################
+
+The perkstore is an online store available in the platform.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
The Perk Store is a digital store available in the platform where users can purchase perks (or whatever goods your organization wants to offer to its users) with their earned funds. 
```

File: docs/UsingPerkStore.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
+.. _UsingPerkStore:
+
+####################
+Using the Perkstore
+####################
+
+The perkstore is an online store available in the platform. 
+You are able to buy the available products or sell goods or services using the cauris as a currency.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
It’s a full-featured marketplace where users can buy (and even sell) products with the tokens stored in their wallet as the digital currency.
```

File: docs/UsingPerkStore.rst [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+How to access to Perkstore
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
How to access to Perk Store
```
```suggestion
How to access to Perkstore
```

File: docs/UsingPerkStore.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access to perkstore from the left navigation. By clicking on the appropriate icon, you will display all the available products in the store.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You can access to Perk Store from the main navigation. By clicking on the shopping cart icon, you will see all the products available for you in the store.
```

File: docs/UsingPerkStore.rst [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+You can see your wallet balance in the toolbar as well as a search bar and a filter button that allow you to look for a specific product.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You can see your wallet balance in the toolbar as well as a search bar and a filter button to help you find a specific product.
```

File: docs/UsingPerkStore.rst [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+|image0|
+
+You can filter the display of articles by:
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You can filter products by:
```

File: docs/UsingPerkStore.rst [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+- Disable products
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
- Disabled products : some products may be temporarily disabled by their seller
```

File: docs/UsingPerkStore.rst [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+- Soldout products
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
- Sold out products : products that are no longer in store
```

File: docs/UsingPerkStore.rst [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+- Created by me
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
- Created by me : the products you offered for sale yourself
```

File: docs/UsingPerkStore.rst [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+
+
+You can see more details about a product by clicking on its title. You can buy the product from this view as well as the perkstore view.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You can see more details about a product by clicking on its title. You can buy the product from this screen as well as the Perk Store screen.
```

File: docs/UsingPerkStore.rst [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+
+|image1|
+
+
+How to buy a product and display your orders
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To buy a product you are interested in, you have to click on the appropriate button |image2| then specify the number of the items to purchase as well as your wallet password if it is not saved in the browser you are using.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
To purchase a product, click on the Buy button button |image2| then specify the number of the items to purchase. Your password may be required unless you checked the “Remember my password” option in your wallet’s preferences.
```

File: docs/UsingPerkStore.rst [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+
+|image3|
+
+After buying the product, You recieve a notification when your order is totally or partially delivered.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
After buying the product, a notification will tell you when your order is totally or partially delivered.
```

File: docs/UsingPerkStore.rst [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+
+|image4|
+
+In the perkstore view a new mention "1 pending" is available for you in the purchased product's card
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
In the Perk Store screen a new mention "1 pending" is available for you in the purchased product's card 
```

File: docs/UsingPerkStore.rst [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+
+After buying the product, You recieve a notification when your order is totally or partially delivered.
+
+|image4|
+
+In the perkstore view a new mention "1 pending" is available for you in the purchased product's card 
+
+|image5|
+
+You can see your order details for this product by clicking in this button.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You can see your order details for this product by clicking on this button.
```

File: docs/UsingPerkStore.rst [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+
+|image6|
+
+To see all orders you can click on "My orders" button from the toolbar of perkstore.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
To see all orders you can click on "My orders" button from the toolbar of the Perk Store.
```

File: docs/UsingPerkStore.rst [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+
+|image7|
+
+|image8|
+
+
+How to create a product
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are allowed to create products from perkstore, you will see the creation card as the first card displayed in the interface.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
If you are allowed to create products , you will see the “Plus” card as the first card displayed in the interface. 
```

File: docs/UsingPerkStore.rst [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+
+
+How to create a product
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are allowed to create products from perkstore, you will see the creation card as the first card displayed in the interface. 
+
+|image9|
+
+By clicking on it, the creation form is displayed.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
By clicking on it, the Product creation form is displayed.
```

File: docs/UsingPerkStore.rst [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+
+|image10|
+
+You have to fill in the mandatory fields which are:
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
Mandatory fields which are:
```

File: docs/UsingPerkStore.rst [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+- The name of the product, 
+- The selling price: The product's price in Cauris
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
- The sell price: The product's price in token currency value
```

File: docs/UsingPerkStore.rst [hunk 21]
```diff
@@ -0,0 +1,106 @@
...
+- The merchant wallet: The user's or space's wallet that will ecieve the funds when the product is purchased, and
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
- The merchant wallet: The user's or space's wallet that will receive the funds when the product is purchased, and
```

File: docs/UsingPerkStore.rst [hunk 22]
```diff
@@ -0,0 +1,106 @@
...
+- The product editor: This field is automatically pre-loaded with the current user account and you have the possibilityto change this.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
- The product editor: This field is automatically pre-loaded with your user account but you can change it if your want to designate another beneficiary for the product sales revenues


.
```

File: docs/UsingPerkStore.rst [hunk 23]
```diff
@@ -0,0 +1,106 @@
...
+
+You can also optionnaly upload images, add a description of the product and select authorized buyers (users or spaces members).
+You may also specify the stock limit and to restrict the number of purchased items by user and per periodicity.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You may also specify a stock limit and to restrict the number of purchased items by user and per periodicity.
```

File: docs/UsingPerkStore.rst [hunk 24]
```diff
@@ -0,0 +1,106 @@
...
+
+|image11|
+
+|image12|
+
+Once your product is sold, you will recieve a notification.
+
+|image13|
+
+You can see the client's order by clicking on the notification or clicking on the following icon |image14| from the perkstore interface.
```
Comment (plamarque) [inline docs/UsingPerkStore.rst]:
```suggestion
You can inspect your clients orders by clicking on the notification or by clicking on the following icon |image14| from the Perk Store interface.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
suggested language and explanatory improvements and fixes

File: docs/UsingPerkStore.rst (hunk 22 above)
Comment (samehaidi) [reply docs/UsingPerkStore.rst]: The product editor has the right to edit the product but doesn't receive funds, the funds are sent to the merchant wallet

File: docs/UsingPerkStore.rst (hunk 9 above)
Comment (samehaidi) [reply docs/UsingPerkStore.rst]: The products that are no longer available in stock.
WDYT?

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**